evalml/tests/automl_tests/test_time_series_split.py

```diff
@@ -60,22 +60,22 @@ def test_time_series_split(max_delay, gap, date_index, X_none, y_none):
 
     answer = [
         (
-            pd.date_range("2020-10-01", f"2020-10-{10 + gap}"),
-            pd.date_range(f"2020-10-{11 - max_delay}", f"2020-10-{17 + gap}"),
+            pd.date_range("2020-10-01", f"2020-10-10"),
+            pd.date_range(f"2020-10-11", f"2020-10-17"),
```

Comment:
Curious: why do these no longer depend on gap and max_delay?

Reply:
They previously had padding that was related to the values of gap and max_delay so that the pipeline had the right amount of historical data to generate predictions on the holdout set. But we're now just passing that data directly in `predict` and `predict_in_sample` so there's no need to pad based on gap and max delay in the data splitter.

Reply:
Thanks for the explanation!